src/CMakeLists.txt

```diff
@@ -532,7 +532,7 @@ if (HAVE_DLADDR AND HAVE_LIBDL)
 	target_link_libraries (gmtlib dl)
 endif (HAVE_DLADDR AND HAVE_LIBDL)
 
-if (MSVC AND FFTW3_FOUND)
+if ((MSVC OR MINGW) AND FFTW3_FOUND)
```

Comment:
This should've been just `if (WIN32 AND FFTW3_FOUND)` as it is platform specific, not compiler/toolchain specific...